packages/patternfly-4/react-core/src/components/AlertGroup/AlertGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from '@patternfly/patternfly/components/AlertGroup/alert-group.css';
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface AlertGroupProps extends Omit<React.HTMLProps<HTMLUListElement>, 'className'> {
+  /** Additional classes added to the AlertGroup */
+  className?: string;
+  /** Alerts to be rendered in the AlertGroup */
+  children?: React.ReactNode;
+  /** Toast notifications are positioned at the top right cornder of the viewport */
+  toast?: boolean;
+}
+
+const AlertGroup: React.FunctionComponent<AlertGroupProps> = ({
+  children = [],
+  className,
+  toast,
+  ...props
+}: AlertGroupProps) => (
+    <ul
+      className={css(styles.alertGroup, className, (toast ? 'pf-m-toast' : ''))}
+      {...props}>
+      {React.Children.toArray(children).map(
+        (Alert: React.ReactNode, index: number) => {
+          return (
+            <li className="pf-c-alert-group__item" key={index}>{Alert}</li>
```

Comment:
`styles.alertGroupItem` available?

Reply:
Doesn't look like this one is available yet either :/